### ISSUE TITLE
[cupertino_icons] Icons for create and create_solid from CupertinoIcons are the same (and buggy)

### ISSUE THREAD

**Issue by joelsongsouzza** [flutter/flutter#106065]
### Steps to reproduce:

1. Open Cupertino Icons [documentation](https://api.flutter.dev/flutter/cupertino/CupertinoIcons-class.html)
2. Find [create](https://api.flutter.dev/flutter/cupertino/CupertinoIcons/create-constant.html) and [create_solid](https://api.flutter.dev/flutter/cupertino/CupertinoIcons/create_solid-constant.html)

The documentation for 'create' says: "[...] This icon is not filled in. [...]", but is not true.

**Expected Result:**
<img width="140" alt="Captura de Tela 2022-06-15 às 18 39 35" src="https://user-images.githubusercontent.com/59066471/173934555-ebfc2c54-e594-4da7-903e-c5a73ba3a47f.png">

**Actual Result:**
<img width="98" alt="Captura de Tela 2022-06-15 às 18 39 53" src="https://user-images.githubusercontent.com/59066471/173934617-a860e1e6-db62-4c24-be70-62b8652c1d72.png">

**Comment by exaby73** [comment]
I can reproduce this issue on stable and master.

<details><summary>Screenshot</summary>

<img width="150" alt="image" src="https://user-images.githubusercontent.com/50555895/174029712-9208575e-1211-4a1e-8c35-1f5acf92ff66.png">

</details>

<details><summary>Code sample</summary>

```dart
import 'package:flutter/cupertino.dart';
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: const MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key? key, required this.title}) : super(key: key);

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  int _counter = 0;

  void _incrementCounter() {
    setState(() {
      _counter++;
    });
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            const Text(
              'You have pushed the button this many times:',
            ),
            Text(
              '$_counter',
              style: Theme.of(context).textTheme.headline4,
            ),
          ],
        ),
      ),
      floatingActionButton: FloatingActionButton(
        onPressed: _incrementCounter,
        tooltip: 'Increment',
        child: const Icon(CupertinoIcons.create),
      ),
    );
  }
}
```

</details>

<details><summary><code>flutter doctor -v</code> (Stable)</summary>

```
[✓] Flutter (Channel stable, 3.0.2, on macOS 12.4 21F79 darwin-arm, locale en-US)
    • Flutter version 3.0.2 at /Users/nabeelparkar/fvm/versions/stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision cd41fdd495 (8 days ago), 2022-06-08 09:52:13 -0700
    • Engine revision f15f824b57
    • Dart version 2.17.3
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc4)
    • Android SDK at /Users/nabeelparkar/Library/Android/sdk/
    • Platform android-33, build-tools 33.0.0-rc4
    • ANDROID_SDK_ROOT = /Users/nabeelparkar/Library/Android/sdk/
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • CHROME_EXECUTABLE = /Applications/Brave Browser.app/Contents/MacOS/Brave Browser

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] IntelliJ IDEA Ultimate Edition (version 2022.1.2)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 68.1.4
    • Dart plugin version 221.5787.37

[✓] VS Code (version 1.68.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.42.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-arm64   • macOS 12.4 21F79 darwin-arm
    • Chrome (web)    • chrome • web-javascript • Brave Browser 102.1.39.122

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

</details>

<details><summary><code>flutter doctor -v</code> (Master)</summary>

```
[✓] Flutter (Channel master, 3.1.0-0.0.pre.1266, on macOS 12.4 21F79 darwin-arm, locale en-US)
    • Flutter version 3.1.0-0.0.pre.1266 at /Users/nabeelparkar/fvm/versions/master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision e1d9adf483 (2 hours ago), 2022-06-15 21:56:04 -0700
    • Engine revision f8c0dc87bc
    • Dart version 2.18.0 (build 2.18.0-189.0.dev)
    • DevTools version 2.14.0

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc4)
    • Android SDK at /Users/nabeelparkar/Library/Android/sdk/
    • Platform android-33, build-tools 33.0.0-rc4
    • ANDROID_SDK_ROOT = /Users/nabeelparkar/Library/Android/sdk/
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • CHROME_EXECUTABLE = /Applications/Brave Browser.app/Contents/MacOS/Brave Browser

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] IntelliJ IDEA Ultimate Edition (version 2022.1.2)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 68.1.4
    • Dart plugin version 221.5787.37

[✓] VS Code (version 1.68.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.42.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-arm64   • macOS 12.4 21F79 darwin-arm
    • Chrome (web)    • chrome • web-javascript • Brave Browser 102.1.39.122

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

</details>

**Comment by HansMuller** [comment]
CC @jmagman

**Comment by jason-simmons** [comment]
Is this icon actually included in the `CupertinoIcons.ttf` font?

I tried dumping the glyphs in the font.  There is a `square_pencil_fill` glyph which is mapped to both `CupertinoIcons.create` and `CupertinoIcons.create_solid`.

But I can't find a version of square_pencil that is not filled.

@xster

**Comment by xster** [comment]
umm something is indeed odd. It comes from https://github.com/xster/framework7-icons/blob/master/build/existing_map.py#L182 and the original icons are https://github.com/xster/framework7-icons/blob/master/src/sf/square_pencil.svg and https://github.com/xster/framework7-icons/blob/master/src/f7/square_pencil_fill.svg which look correct. Which makes it sound like something in the font forge process broke the glyphs.

**Comment by joelsongsouzza** [comment]
Any updates on this?

**Comment by tschiekdev** [comment]
After more than a year still no correction of the icon? Why?

**Comment by HansMuller** [comment]
@MitchellGoodwin - @tschiekdev and #116594 have a point. Per https://github.com/flutter/flutter/issues/106065#issuecomment-1163353885, maybe something in the icon production process isn't quite right.

**Comment by ercantomac** [comment]
Any updates on this?

**Comment by ricardoboss** [comment]
@HansMuller any way we can contribute to the icon production process to fix this? Is there documentation for the process? Is someone working on it already?

**Comment by LongCatIsLooong** [comment]
Should be fixed in CupertinoIcons 1.08
